lib/emmet-jsx-css-modules.js

```diff
@@ -58,7 +58,7 @@ export default {
 
       filters.add('jsx-css-modules', (tree) => {
         tree.children.forEach((item) => {
-          item.start = item.start.replace(/className="(.*?)"/, replFn);
+          item.start = item.start.replace(/className=[",'](.*?)[",']/, replFn);
```

Comment:
Right you are. Silly me. I use that above version and since I never type 

```
className=,name,
```

i've never noticed it was wrong, thanks. Ill fix that now.